src/assets/arrow.svg

```diff
@@ -0,0 +1 @@
+<svg width="51" height="51" fill="none" xmlns="http://www.w3.org/2000/svg"><circle opacity=".4" cx="25.395" cy="25.427" r="25" fill="#DFEBF7"/><path d="M24.483 19.47h0l-1.075 1.125s0 0 0 0l4.39 4.832-4.39 4.831a.85.85 0 000 1.126h0a.717.717 0 001.075 0h0l4.9-5.395h.001a.856.856 0 000-1.125l-4.901-5.395z" fill="#000" stroke="#000" stroke-width=".4"/></svg>
```

Comment:
Can we use `IconRight` from the kit rather than adding this? if we want a slightly thicker icon to match the design we should be able to do:
```javascript
<IconRight
        css={`
          stroke: currentColor;
          stroke-width: 1px;
        `}
      />
```

To pad the path with a bit of extra weight.